lib/active_record/connection_adapters/oracle_enhanced/schema_statements.rb

```diff
@@ -504,6 +505,22 @@ def default_trigger_name(table_name)
           "#{table_name.to_s[0,table_name_length-4]}_pkt"
         end
 
+        def rebuild_primary_key_index_to_default_tablespace(table_name, options)
+          tablespace = default_tablespace_for(:index)
+
+          return unless tablespace
+
+          index_name = Base.connection.select_value(
+            "SELECT index_name FROM all_constraints
+                WHERE table_name = #{quote(table_name.upcase)}
+                AND constraint_type = 'P'
+                AND owner = SYS_CONTEXT('userenv', 'current_schema')")
+
+          return unless index_name # schema_migrations.id does not have index name
```

Comment:
I do not think we need this condition since `all_constraints` returns the index for the primary key.

```sql

SQL> SELECT const.table_name,const.index_name, col.column_name
  2  FROM all_constraints const, all_cons_columns col
  3  WHERE const.table_name = 'SCHEMA_MIGRATIONS'
  4  AND const.constraint_type = 'P'
  5  AND const.owner = SYS_CONTEXT('userenv', 'current_schema')
  6  AND const.owner = col.owner
  7  AND const.constraint_name = col.constraint_name
  8  /

TABLE_NAME           INDEX_NAME           COLUMN_NAME
-------------------- -------------------- --------------------
SCHEMA_MIGRATIONS    SYS_C00125899        VERSION

SQL> desc SCHEMA_MIGRATIONS
 Name                                      Null?    Type
 ----------------------------------------- -------- ----------------------------
 VERSION                                   NOT NULL VARCHAR2(255 CHAR)

SQL>
```